pegen/pegen.c

```diff
@@ -517,11 +526,6 @@ run_parser(struct tok_state *tok, void *(start_rule_func)(Parser *), int mode)
     if (!p->arena) {
         goto exit;
     }
-
-    if (fill_token(p) < 0) {
-        goto exit;
-    }
-
```

Comment:
Curious why this can be deleted?

Reply:
This actually has to be deleted, since at the time this is called `start_rule` has not been called yet, which means that `init_keywords` hasn't been called as well. Thus calling `fill_token` here would segfault.